fms/models/gpt_bigcode.py

```diff
@@ -58,20 +58,19 @@ def __init__(self, config: GPTBigCodeConfig):
 
     def forward(
         self,
-        x: torch.LongTensor,
+        x: torch.Tensor,
         *,
         mask: Optional[torch.Tensor] = None,
-        position_ids: Optional[torch.LongTensor] = None,
+        position_ids: Optional[torch.Tensor] = None,
         past_key_value_state: Optional[
             Tuple[
-                torch.FloatTensor,
+                torch.Tensor,
```

Comment:
Why switch this to just a tensor? Isn't it always a float tensor?

Reply:
see above